contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-07
+title: You can now organize your PostHog data
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
+
+This is the first step in a continuous effort to help make organizing your data easier:
+
+- Creating definitions for events and event properties
+- Managing data on the fly while building insights
+- New UI (Data Management tab and revamped definitions page)
+- Verifying trusted definitions
+- Categorizing your definitions with tags
+- Giving custom names and descriptions to events and properties
+- Making the right events and properties discoverable with event metrics
```

Comment:
```suggestion
This is the first step in a continuous effort to help make organizing your data easier. With the new Data Management tab and UI you can:

- Create definitions for events and event properties
- Manage data on the fly while building insights
- Verify trusted definitions which are prioritzed
- Organize your definitions with tags
- Give custom names and descriptions to events and properties
- Make the right events and properties discoverable with event metrics
```

Reply:
Tweaked this section a little just to make flow a little better. I think it would be a good idea to have a screenshot of the Data Management tab below this list. Could be a short gif/video if that's practical?

Reply:
Yes! great idea